packages/oui-navbar/src/menu/navbar-menu.component.js

```diff
@@ -10,6 +10,7 @@ export default {
         headerBreadcrumb: "@?",
         headerClass: "@?",
         headerTitle: "@?",
+        headerLink: "@?",
```

Comment:
To keep consistency with all attributes about creating links, we should rename it `headerHref` :)